src/test/scala-2.12/org/apache/james/gatling/simulation/Injection.scala

```diff
@@ -9,8 +9,8 @@ abstract sealed class UsersDensity {
   def injectDuring(givenDuring: FiniteDuration): OpenInjectionStep
 }
 case class UsersPerHour(nb: Double) extends UsersDensity {
-  private def usersPerSecForDuration(givenDuring: FiniteDuration): Double = nb * givenDuring.toSeconds / 1.hour.toSeconds
-  override def injectDuring(givenDuring: FiniteDuration): ConstantRateOpenInjection = constantUsersPerSec(usersPerSecForDuration(givenDuring)) during givenDuring
+  private def usersPerSecForDuration(): Double = nb / 1.hour.toSeconds
```

Comment:
Then, you will inject the same amount of user whatever the duration is?

Reply:
It depends what you want here exactly... I put it back to what it was before, because anyway the previous math was wrong

Maybe it should be more then `nb / givenDuring.toSeconds()` then?

Number of users I want to inject in the total period of time?

Reply:
I've just tried it, I think it's good, just drop the `()`

Reply:
actually I rechanged it to `nb / givenDuring.toSeconds`

the function name speaks for itself. We want the uers per sec for duration. `nb` is the total number of users we want to inject for the duration of the simulation, `givenDuring` is the total duration of the simulation

So users injected per second for the total duration should be `nb / givenDuring.toSeconds`, it's more logical this way.